HACKING.md

```diff
@@ -188,6 +188,22 @@ EOF
 oc create secret generic aws-fcos-kola-bot-config --from-file=config=/path/to/kola-secret
 ```
 
+### Allocating S3 storage
```

Comment:
Let's add an `[OPTIONAL]` tag to this header. In fact, the pipeline works fine without any AWS account at all. In that case, builds are kept in the PVC, and one can use `simple-httpd` to access them.

Not practical for long-term storage of course, but for local testing and iterating, it works well.

Reply:
Done.